pria_lifechem/models/test_demo.py

```diff
@@ -0,0 +1,110 @@
+import argparse

```

Comment:
Please mention this new test code and what it does in `pria_lifechem/models/README.md`.